core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -205,9 +207,6 @@
   CoordinatorLayout snackbarRoot;
   @BindView(R2.id.fullscreen_video_container)
   ViewGroup videoView;
-
-  @Inject
```

Comment:
why did this move from the other injected fields?

Reply:
I just did ⌘+⌥+L to reformat the code after importing Square's code style to reformat the code. I think this move because it's protected.

Reply:
bring up the reformat dialog for java files, for me that is `ctrl+alt+shift+l`, and untick `rearrange entries`

Reply:
My reformat dialog had rearrange entries checked. :p

Reply:
_untick_

Reply:
Done. I think we should also rearrange the code in our pre-commit hook.

Reply:
There is no rearrange for kotlin so I don't rate it highly